integrationTests/chainSimulator/relayedTx/sovereignRelayedTx_test.go

```diff
@@ -0,0 +1,58 @@
+package relayedTx
+
+import (
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/stretchr/testify/require"
+
+	sovChainSimulator "github.com/multiversx/mx-chain-go/cmd/sovereignnode/chainSimulator"
+	"github.com/multiversx/mx-chain-go/config"
+	testsChainSimulator "github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+)
+
+func TestRelayedV3WithSovereignChainSimulator(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	runRelayedV3TestsWithChainSimulator(t, sovereignChainSimulator)
+}
+
+func sovereignChainSimulator(
+	t *testing.T,
+	alterConfigsFunction func(cfg *config.Configs),
+) testsChainSimulator.ChainSimulator {
+	roundDurationInMillis := uint64(6000)
+	roundsPerEpochOpt := core.OptionalUint64{
+		HasValue: true,
+		Value:    roundsPerEpoch,
+	}
+
+	cs, err := sovChainSimulator.NewSovereignChainSimulator(sovChainSimulator.ArgsSovereignChainSimulator{
+		SovereignConfigPath: defaultPathToSovereignConfig,
+		ArgsChainSimulator: &chainSimulator.ArgsChainSimulator{
+			BypassTxSignatureCheck:   true,
+			TempDir:                  t.TempDir(),
+			PathToInitialConfig:      defaultPathToInitialConfig,
```

Comment:
How does this work without path to sovereign config?

Reply:
sovereign config path is at line 36